src/main/scala/pt/tecnico/dsi/neutron/models/package.scala

```diff
@@ -1,10 +1,16 @@
 package pt.tecnico.dsi.neutron
 
+import java.net.Inet4Address
```

Comment:
Codacy found an issue: [Use brackets to group imports from the same package](https://app.codacy.com/gh/ist-dsi/scala-neutron-client/pullRequest?prid=5802728)